include/dbglogger.h

```diff
@@ -37,6 +37,11 @@ extern "C" {
 //  TCP example string    "tcp:192.168.1.123:18194"
 //  File example string   "file:/dev_hdd0/temp/app.log"
 
+#ifdef (__USE_TTY_LOG__)
+#define TTY_LOGGER puts
+#else
+#define TTY_LOGGER(...)
+#endif

```

Comment:
No, I mean you should add a `TTY_LOGGER` option to the LOGGER_MODES `typedef enum`

then handle the case inside the switch(loggerMode) of `dbglogger_printf`